make/include/defaults

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+: "${GIT_ROOT:=$(git rev-parse --show-toplevel)}"
+source "${GIT_ROOT}/.envrc"
+
+: "${CF_NAMESPACE:=cf}"
+: "${DOMAIN:=cf-dev.io}"
+: "${UAA_CF_IDENTITY_ZONE:=cf}"
```

Comment:
Isn't this always the CF namespace, as set in [uaa-create-user](https://github.com/SUSE/scf/blob/d4858f0d424fbd008f34d9a9ef3e04ad9256cc1f/src/scf-release/jobs/uaa-create-user/templates/run.erb#L69)?

Reply:
Cool, thanks for pointing out. I wasn't sure if they always matched.